Create checkbox (#27)
magnetis/astro: docs/controls-toggles.mdx

---
name: Controls & Toggles
---

import "../src/index.css";
import { Playground } from "docz";

# controls & toggles

## checkboxes

> Checkboxes are used when the user can select multiple options from a list or when a single and specific action is required like, for example, reading a contract.

Reviewer: May I suggest a change in copy? (the designers already know I always suggest changes lol)

```suggestion
Checkboxes are used when a user might select multiple options from a list, or when a single specific action is required - like reading a contract.
```

Author: Done on commit https://github.com/magnetis/astro/pull/27/commits/40b77eaa23bb117dc49a75bf7b6bc7d8157cfbf0 .

Mouseover to view `:hover` state.

<Playground>
  <div className="checkbox">
    <input type="checkbox" id="checkbox1" />
    <label htmlFor="checkbox1">Checkbox</label>
  </div>

  <div className="checkbox">
    <input type="checkbox" id="checkbox2" checked />
    <label htmlFor="checkbox2">Checked</label>
  </div>

>   <div className="checkbox checkbox--indeterminated">

Reviewer: Same thing with "indeterminate";

Author: Done on commit https://github.com/magnetis/astro/pull/27/commits/3d3459e987c8d4aa3a3e501872c1d3acf1953d4a .

    <input type="checkbox" id="checkbox3" checked />

>     <label htmlFor="checkbox3">Indeterminated</label>

Reviewer: Same thing with "indeterminate";

Author: Done on commit https://github.com/magnetis/astro/pull/27/commits/3d3459e987c8d4aa3a3e501872c1d3acf1953d4a .
